index.html

```diff
@@ -219,12 +219,136 @@ <h2 class="destinations__title">The best destinations</h2>
+        <ul class="reviews__list">
+          <li class="reviews__item">
+            <div class="reviews__image-wrap">
+              <img
+                src="./img/Resort&Spa.svg"
+                alt="Resort&Spa"
+                class="reviews__image"
+              />
+              <p class="reviews__image-title">From 250 USD</p>
+            </div>
+            <div class="reviews__wrapper">
+              <p class="reviews__subtitle">Ubud Bali Resort&SPA</p>
+              <p class="reviews__rating">9,4</p>
+            </div>
+            <div class="reviews__wrapper wrap-margin">
+              <p class="reviews__desc">Bali, Indonesia</p>
+              <p class="reviews__counter">1324 reviews</p>
+            </div>
+            <div class="reviews__guests">
+              <svg class="reviews__guests-account">
+                <use xlink:href="./img/Sprites.svg#Account"></use>
+              </svg>
+              <div class="reviews__guests-info">
+                <p class="reviews__guests-name">Hanna Ivanova</p>
+                <p class="reviews__guests-country">
+                  <img
+                    src="./img/ukraine.svg"
+                    alt="Ukraine"
+                    class="reviews__country"
+                  />
+                  Ukraine
+                </p>
+              </div>
+            </div>
+            <p class="reviews__text">
+              Great location, really pleasant and clean rooms, but the thing
+              that makes this such a good place to stay are the staff. All of
+              the people are incredibly helpful and generous with their time and
+              advice.
+            </p>
+          </li>
+          <li class="reviews__item">
+            <div class="reviews__image-wrap">
+              <img src="./img/Hostel.svg" alt="Hostel" class="reviews__image" />
+              <p class="reviews__image-title">From 40 USD</p>
+            </div>
+            <div class="reviews__wrapper">
+              <p class="reviews__subtitle">King Kong Hostel</p>
+              <p class="reviews__rating">8,9</p>
+            </div>
+            <div class="reviews__wrapper wrap-margin">
+              <p class="reviews__desc">Rotterdam, Netherlands</p>
+              <p class="reviews__counter">2016 reviews</p>
+            </div>
+            <div class="reviews__guests">
+              <svg class="reviews__guests-account">
+                <use xlink:href="./img/Sprites.svg#Account"></use>
+              </svg>
+              <div class="reviews__guests-info">
+                <p class="reviews__guests-name">Adam Smith</p>
+                <p class="reviews__guests-country">
+                  <img src="./img/usa.svg" alt="Usa" class="reviews__country" />
+                  USA
+                </p>
+              </div>
+            </div>
+            <p class="reviews__text">
+              I've been in this hostel for a month and here is the best hostel
```

Comment:
You should keep paragraphs inside individual `<p>`, it's 3 `<p>` here, try to avoid `<br>`